cmd/main.go

```diff
@@ -97,6 +97,7 @@ func addonManager(db *gorm.DB, adminChanged chan int) (mgr manager.Manager, err
 func main() {
 	log.Info("Started", "settings", Settings)
 	var err error
+	syncHelper := model.SyncHelper{}
```

Comment:
In order to keep the same mutex in Gin.based API and backend Import manager, it was needed add the mutex to "top level" of the application, which maybe results in slightly bigger change than expected.